incubator/hnc/pkg/kubectl/create2.go

```diff
@@ -0,0 +1,52 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var create2Cmd = &cobra.Command{
+	Use:   "create2 -n parent child1 [child2 ...]",
+	Short: "Creates subnamespaces under the given parent.",
+	Args:  cobra.RangeArgs(1, 10),
```

Comment:
Does `k create ns` allow multiple args? If not, we shouldn't either, IMO?

Reply:
No. sg.